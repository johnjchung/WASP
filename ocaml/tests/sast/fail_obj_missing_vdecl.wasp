
int a = 11;
int b = 13;
int c = 12;

Endpoint testing(GET:){
}
