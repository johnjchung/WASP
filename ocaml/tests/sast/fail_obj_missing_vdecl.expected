Fatal error: exception Wasp.SyntaxError(1, 63, "}")
